src/test/tasks.yaml

```diff
@@ -43,12 +43,19 @@ tasks:
             address: demo.admin.uds.dev/status/410
             code: 410
 
-      - description: Verify the tenant app is accessible
+      - description: Verify the tenant app muli port 8080 is accessible
         wait:
           network:
             protocol: https
-            address: demo.uds.dev/status/202
-            code: 202
```

Comment:
This removed the previous tenant demo app wait it looks like. I think having a test for multi-port is valuable but wonder if we could just do it as part of one of our other existing tests so that we aren't adding more pods to the scope of our test (we're not far off from the resource limit for the runner 😢). Maybe we could replace the current tenant app check with this one? Ideally it would also be great to keep this to a single image/container that is listening on two ports but even reducing scale by one less container/pod would be good.

Reply:
Ooops. I didn't mean to remove that original one. Was only supposed to be additive. I originally tried to use httpbin for this multiport test, but it wouldn't work because it only listens on port 80. I needed a container that listened on multiple ports to force it to need to create multiple netpols for the same pod with the same pod selector. 

I think http-echo could work for the other app tests? in place of httpbin?

Reply:
So I re worked this a bit. Its back to just the app-tenant manifests but exercises multi port. I can't recreate the admin tests with http-echo and I can't recreate the mutli-port tests using httpbin. I did cut resource usage in half on everything and that all works? Maybe thats acceptable?